dashdotdb/models/deploymentvalidation.py

```diff
@@ -0,0 +1,60 @@
+from dashdotdb.models.base import db
+
+
+class Token(db.Model):
+
+    __tablename__ = 'token'
```

Comment:
we should probably rename the https://github.com/app-sre/dashdotdb/blob/master/dashdotdb/models/imagemanifestvuln.py file to a more generic name and add the tables you need there.

Reply:
maybe! we could also keep each piece distinct. locally, i'm working with extend_existing for the table re-use.

ex:
```
class Namespace(db.Model):                                                      
                                                                                
    __tablename__ = 'namespace'                                                 
    __table_args__ = {'extend_existing':True}                                   
                                                        
````

Reply:
I see... that works :+1: 
Maybe I bit harder to maintain, but no biggies I guess.

Reply:
i suspect we'll be iterating on the dashdotdb metric format as we go on.. so initially, i leaned toward keeping distinct classes + schema, allowing for non-destructive overlap. thought was: there may be some convenience if we package up dddb related pieces with each service/operator or as a sub-path, for a service, in app-interface.  we had a more generic (external-to-a-i) model like this for the observability/dashboarding bits, and  eventually blended that into the app-interface tenant model.. given it's early the single-generic model for simplicity is a good suggestion.

Reply:
hmm. as i dive through this, i've been running into just a cascade of mapping problems if i want to keep an inheritance model so that multiple models can exist for a single table, i'll need to use: single table inheritence.. so, i am rewriting the classes a bit with this:  https://docs.sqlalchemy.org/en/13/orm/inheritance.html#single-table-inheritance
 this should still work then for the idea that tenants can (eventually) self-service extend tables for their own app-sre managed services.